### PR TITLE
Made documentation website with mkdocs.

### LINKED ISSUE #13: Add documentation Website
### Please provide all the data here.

We should add a documentation website similar to https://surajincloud.github.io/kubectl-eks/ using the https://squidfunk.github.io/mkdocs-material/ theme.

Once the doc site is added, every PR should be merged only if documentation is updated (if applicable)

### PR DESCRIPTION
**Pull request by adityasimant**
## Fixes Issue
Fixes #13 

## Changes made

Installed mkdocs
Added mkdocs.yml 
Added index.md, usage.md, install.md and contributing.md 

## Note to reviewers
This documentation will be improved as our projects develops.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+site_name: awsctl
+site_author: Suraj Narwade
```
[the remark this answers is not in the document]
Comment (adityasimant) [reply mkdocs.yml:2]: as site name?

[the remark this answers is not in the document]
Comment (surajnarwade) [reply mkdocs.yml:2]: site_author

File: docs/contributing.md
Comment (surajnarwade) [inline docs/contributing.md:1]: we don't need this file as we already have this in the repo. maybe we can add hyperlink somewhere which will point to https://github.com/surajincloud/awsctl/blob/main/CONTRIBUTING.md

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Check out install page for [installation](./install) instructions.
+
+## Usage
+
+Check out Usage page for [Usage](./usage) instructions.
+
+## Contributing
+
+Check out Contributing page for [Contributing](./contributing) instructions.
```
Comment (surajnarwade) [inline docs/index.md:16]: just point to https://github.com/surajincloud/awsctl/blob/main/CONTRIBUTING.md

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        name: Switch to dark mode
+    - scheme: slate
+      primary: indigo
+      accent: indigo
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+
+repo_name: surajincloud/awsctl
+repo_url: https://github.com/surajincloud/awsctl
```
Comment (surajnarwade) [inline mkdocs.yml:20]: add newline at the end

Comment (surajnarwade) [reply mkdocs.yml:20]: @adityasimant ^^

**Comment by surajnarwade** [comment]
@adityasimant LGTM, since it's in Draft, I can't merge this, can you convert to review mode if it is completed?

**Review by surajnarwade — APPROVED** [review]

**Pull request merged**